lang/da.php

```diff
@@ -0,0 +1,143 @@
+<?php
+
+$lang['title'] = 'TecArt Ticket System';
+
+$lang['foot_note'] = 'Ticket-Web Interface &copy; by TecArt GmbH';
+
+$lang['custom_number']  = 'Kundenummer';
+$lang['password']       = 'Kodeord';
+$lang['login']          = 'Brugernavn';
+$lang['headline_standard'] = 'Ticket-Web Interface';
+$lang['headline']       = 'TecArt Ticket-Web Interface';
+$lang['logout']         = 'Log ud';
+$lang['view']           = 'Vis';
+
+$lang['show_current']       = 'Aktuelle billetter';
```

Comment:
I'm not at all fluid at danish but shouldn't that be rather 'sager' than 'billeter'?

Reply:
Yeah that would definently make more sense, I just did not fully understand the context. Want me to change it?

Reply:
Yeah sure :) If you're okay with German(or browser auto-translated English) you could take look at https://tts.tecart-system.de/ for a reference.